README.md

```diff
@@ -199,8 +199,10 @@ welcome and encouraged to join.
 
 Please use the following to reach members of the community:
 
-- Slack: Join our [Slack channel](https://ceph-storage.slack.com) to discuss
-  anything related to this project. You can join the Slack by this [invite
-  link](https://bit.ly/40FQu7u)
+- Slack: Join the
+  [#ceph-csi](https://ceph-storage.slack.com/archives/C05522L7P60) channel 
+  on the [ceph Slack](https://ceph-storage.slack.com) to discuss anything
+  related to this project. You can join the Slack by this
+  [invite link](bit.ly/ceph-slack-invite)
```

Comment:
Thanks!

For any others that wonder how to verify the link, you can add a `+` to the URL, like `https://bit.ly/ceph-slack-invite+` to see where it points to.